WordPressKit/WordPressKit/BlogJetpackSettingsServiceRemote.swift

```diff
@@ -119,6 +119,45 @@ public class BlogJetpackSettingsServiceRemote: ServiceRemoteWordPressComREST {
                                  })
     }
 
+    public func fetchTimeZoneList(success: @escaping ([String: [String: String]]) -> Void, failure: @escaping (Error) -> Void) {
```

Comment:
For this particular setting as well as the other ones you will be working on, they should be available for WordPress.com sites as well as Jetpack sites, so the right place for this method would be `BlogServiceRemoteREST`